website/sidebarsApi.js

```diff
@@ -173,6 +173,11 @@ export default {
           label: "zkVM technical specification",
           id: "zkvm/zkvm-specification",
         },
+        {
+          type: "link",
+          label: "Awesome risc0",
+          href: "https://github.com/inversebrah/awesome-risc0",
```

Comment:
I feel like this is a little bit of a strange territory... If we create this link, we're reliant on this individual maintaining this list on our behalf. Why not have a page on our own page listing these projects and others? I think it would be better for maintainability if we have a page that features projects that we maintain